wrappers.js

```diff
@@ -25,27 +26,31 @@ class ConnectionWrapper {
     * @func
     * @param {object} event - The event object created when parsing the incoming messages.
     * @param {string} message - The message you wish to reply with.
+    * @param {string} [background=null]
+    * @param {boolean} [rainbow=false]
+    * @param {string} [style=null]
     */
-    reply(event, message) {
-        if (event.target === this.bot.nickname) {
-            this.privmsg(event.source.nick, message);
-        } else {
-            this.privmsg(event.target, message);
-        }
+    reply(event, message, background=null, rainbow=false, style=null) {
+        let isPRIVMSG = event.target === this.bot.nickname;
+
+        this.privmsg(isPRIVMSG ? event.source.nick : event.target, message, background, rainbow, style);
     }
 
     /**
     * @func
     * @param {string} target - The user or channel you wish to send a PRIVMSG to.
     * @param {string} message - The message you wish to send.
+    * @param {string} [background=null]
+    * @param {boolean} [rainbow=false]
+    * @param {string} [style=null]
     */
-    privmsg(target, message) {
+    privmsg(target, message, background=null, rainbow=false, style=null) {
```

Comment:
I think it'd be better for plugins/root to add colours to the message before calling message functions with it hence the extra three arguments wouldn't be necessary.

Reply:
Yeah, it would be nice if colours were provided as a util. Maybe something like
```javascript
const colors = require('../utils/colors.js');
colors("some text").red().bold();
```

Reply:
I wouldn't know how to do that kind of function, with the chained functions